h2/src/main/org/h2/table/Table.java

```diff
@@ -1242,5 +1244,13 @@ public void setHidden(boolean hidden) {
     public boolean isMVStore() {
         return false;
     }
+    
```

Comment:
I moved this up from TableView because it's awkward to cast back and forth from TableView and Table.